vpype/text.py

```diff
@@ -210,6 +227,7 @@ def text_block(
             left alignment)
         line_spacing: line spacing (default: 1.0)
         justify: should the text be justified (default: False)
+        hyphenate: wrapped text is hyphenated with the given language (default: None)
```

Comment:
It would be great to indicate what format(s) are acceptable for the language, and provide a couple of examples.